todo-backend/todo-backend-bootable-jar.yaml

```diff
@@ -0,0 +1,43 @@
+# This configuration file can be used to build and deploy the todo-backend
+# quickstart on OpenShift with the Helm Chart for WildFly.
+build:
+  uri: https://github.com/wildfly/quickstart.git
+  ref: todo-backend
```

Comment:
this is a wrong branch

setting it right might be tricky though :( it appears we would need to change it for each release